npm/ng-packs/packages/theme-shared/src/lib/components/card/card.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, ContentChild, Input } from '@angular/core';
+import { CardBodyComponent } from './card-body.component';
+import { CardTitleComponent } from './card-title.component';
+
+@Component({
+  selector: 'abp-card',
+  template: `
+  <div class="card" [ngClass]="class" [ngStyle]="style" >
+  <div class="card-title" *ngIf="cardTitleTemplate">
```

Comment:
card-title component should constain 
` <div class="card-title" *ngIf="cardTitleTemplate">
    <ng-content select="abp-card-title"></ng-content>
  </div>`
  that  or it should be removed.